package/build_boards_manager_package.sh

```diff
@@ -135,11 +126,15 @@ cat $srcdir/package/package_esp8266com_index.template.json | \
 
 # Use Github API token, if available
 curl_gh_token_arg=()
-if [ ! -z "$CI_GITHUB_API_KEY" ]; then
+if [ -z "$CI_GITHUB_API_KEY" ]; then
+    echo "curl: API key not present, exit with no error"
+    exit 0
```

Comment:
I think that not preparing the package and exiting with 0 exit code is a bad behavior for the script. If I run it locally, it should either generate a package or exit with an error.

Skipping this script if the API key is not set should be handled by `if: env(CI_GITHUB_API_KEY) IS present` in travis yml. If you have observed that that does not work, it should be fixed there, I think.

Reply:
Nevermind, I see now the link to the failed job in gitter. It seems that the condition in the yml does not work correctly, as variables get removed after the job has started. I think we need some other condition there, like `fork = false`...

Reply:
No more condition in yml.
* In a PR, package script is verbose and does not fail whether or not curl is failing.
* In a branch, curl failing makes the job failed.